Makefile

```diff
@@ -15,7 +18,20 @@ clean:
 	rm -rf src/__pycache__ src/*/__pycache__ src/*/*/__pycache__
 	rm -rf src/*.egg-info src/*/*.egg-info src/*/*/*.egg-info
 
-ci-build: clean
+readme:
+	python docs/document_readme.py
```

Comment:
Is this right or should it be `update_readme.py`

Reply:
Good point! Fixed